.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 *.log
+Cargo.lock
```

Comment:
@Kethku I am curious why we keep Cargo.lock. It does help give more stable builds.

Reply:
Cargo lock makes builds way more consistent. I would prefer to keep this in unless a cogent argument can be made not to

Reply:
Yes, I understand now.